detection-rules/urlhaus_malicious_dom_in_body_or_pdf.yml

```diff
@@ -23,28 +26,32 @@ source: |
         and .href_url.domain.root_domain in $abuse_ch_urlhaus_domains_trusted_reporters
     )
     or any(attachments,
-           .file_type == "pdf"
-           and any(file.explode(.),
-                   any(.scan.pdf.urls,
-                       // filter potentially known good domains
-                       // prevents FPs on entries such as drive[.]google[.]com, or
-                       // other accidental pushes to the List
-                       .domain.domain not in $free_file_hosts
-                       and .domain.domain not in $free_subdomain_hosts
-                       and .domain.domain not in $tranco_1m
-                       and .domain.domain not in $umbrella_1m
-                       and 
+          .file_type == "pdf"
+          and any(file.explode(.),
+                  any(.scan.pdf.urls,
+                      // filter potentially known good domains
+                      // prevents FPs on entries such as drive[.]google[.]com, or
+                      // other accidental pushes to the List
+                      (
+                        .domain.domain not in $free_file_hosts
+                        or .domain.root_domain not in $free_file_hosts
+                      )
```

Comment:
since this is a not won't this always be true? should this be an and instead like the others in this block?